contributors/guide/coding-conventions.md

```diff
@@ -36,11 +36,11 @@ description: |
 ## Testing conventions
 
   - All new packages and most new significant functionality must come with unit tests.
-  - Table-driven tests are preferred for testing multiple scenarios/inputs. For an example, see [TestNamespaceAuthorization](https://git.k8s.io/kubernetes/test/integration/auth/auth_test.go).
+  - Table-driven tests are preferred for testing multiple scenarios/inputs. For an example, see [TestNamespaceAuthorization](https://cs.k8s.io/?q=func%20TestNamespaceAuthorization&i=nope&files=&excludeFiles=&repos=).
```

Comment:
I think this could be confusing sending readers to the search page. Is there a reason to change it from the current actual location? There is a redirect with using git.k8s.io, but that is fine and expected. So if anything, I would go directly to the GitHub location rather than sending them to code search.

Reply:
> I think this could be confusing sending readers to the search page. Is there a reason to change it from the current actual location?

Reasoning was to improve user experience.  With code search link end user gets a direct link to TestNamespaceAuthorization function quicker. As it is today ( link to git.k8s.io, without direct link to the function ) it requires end user to scroll down and find this function. I could replace with direct git.k8s.io link to the line where TestNamespaceAuthorization is defined ( but would require to update the doc each time )

As a plus end user learn the existence of cs.k8s.io website which is very useful for contibutors, i believe

Reply:
Updated commit with a github permalink instead.